tests/python/pants_test/backend/jvm/tasks/test_ivy_utils.py

```diff
@@ -26,6 +26,15 @@
 from pants_test.test_base import TestBase
 
 
+# The V2 test Pytest runner strips the source root, but V1 keeps it.
```

Comment:
If this is a matter of how `resources` are loaded from the python path, we should almost certainly be consistent between the two runners, as it is effectively a public API at this point.

And if this is a matter of how `files` targets are included in the workdir/chroot of the test: ditto.

Reply:
Yes, this is matter of how they're loaded. I think you're right that we need to somehow make this consistent, which is going to take a little more work.

I reverted these changes and will address it in a dedicated PR.